index.html

```diff
@@ -3,115 +3,142 @@
+      <section class="tile">
+        <div class="tile__text">
+          <h3 class="tile__header">Your Users</h3>
+          <p class="tile__description">
+            It takes no time at all to integrate Huddle with your app's
+            authentication solution. This means, once signed in to your app,
+            your users can start chatting immediately.
+          </p>
+        </div>
+        <img
+          class="tile__img"
+          src="/design/images/illustration-your-users.svg"
+          alt=""
+        />
+      </section>
     </main>
     <footer class="footer">
-        <aside class="footer__aside">
-            <h2 class="aside__title">Ready To Build Your Community?</h2>
-            <button class="cta">Get Started For Free</button>
-        </aside>
+      <aside class="aside">
+        <h2 class="aside__title">Ready To Build Your Community?</h2>
+        <button class="cta">Get Started For Free</button>
+      </aside>
       <div class="footer__main">
-        <img class="footer__logo" src="/design/images/logo-inverted.svg" alt="Huddle contact information">
-        <div class="footer__contact">
+        <img
+          class="footer__logo"
+          src="/design/images/logo-inverted.svg"
+          alt="Huddle contact information"
+        />
+        <div class="contact">
           <div class="contact__item contact__address">
-            <img class="contact__icon" src="/design/images/icon-location.svg" alt="postal address:">
+            <img
+              class="contact__icon"
+              src="/design/images/icon-location.svg"
+              alt="postal address:"
+            />
             <adress class="contact__link">
               Lorem ipsum dolor sit amet, consectetur
-              <br>
+              <br />
               adipiscing elit, sed do eiusmod tempor
-              <br>incididunt ut labore et dolore magna aliqua
+              <br />incididunt ut labore et dolore magna aliqua
             </adress>
-            <img class="contact__icon" src="/design/images/icon-phone.svg" alt="Phone:">
-            <a class="contact__link" href="tel:+1-543-123-4567">+1-543-123-4567</a>
-            <img class="contact__icon" src="/design/images/icon-email.svg" alt="Email:">
-            <a class="contact__link" href="mailto:example@huddle.com">example@huddle.com</a>
+            <img
+              class="contact__icon"
+              src="/design/images/icon-phone.svg"
+              alt="Phone:"
+            />
+            <a class="contact__link" href="tel:+1-543-123-4567"
+              >+1-543-123-4567</a
+            >
+            <img
+              class="contact__icon"
+              src="/design/images/icon-email.svg"
+              alt="Email:"
+            />
+            <a class="contact__link" href="mailto:example@huddle.com"
+              >example@huddle.com</a
+            >
           </div>
-          <div class="contact__item contact__learn-more" aria-label="Learn more:">
```

Comment:
Usually `aria-labels` are used for buttons, links or images or something like that.